src/features/element-hiding.js

```diff
@@ -137,17 +137,27 @@ function isDomNodeEmpty (node) {
     })
 
     const visibleText = parsedNode.innerText.trim().toLocaleLowerCase().replace(/:$/, '')
-    const mediaContent = parsedNode.querySelector('video,canvas,picture')
+    const mediaContent = parsedNode.querySelector('video,canvas,embed,object,audio,map')
+    const formContent = parsedNode.querySelector('form,input,textarea,select,option,button')
```

Comment:
Does it make sense to combine mediaContent and formContent as a single selector that we pass through from the config.